src/main/java/com/salesforce/mirus/MirusSourceTask.java

```diff
@@ -211,7 +209,8 @@ private void seekToOffsets(List<TopicPartition> partitionIds) {
   }
 
   private boolean isSkippedRecord(ConsumerRecord<byte[], byte[]> consumerRecord) {
-    TopicPartition topicPartition = new TopicPartition(consumerRecord.topic(), consumerRecord.partition());
+    TopicPartition topicPartition =
```

Comment:
Better to avoid mixing code changes with reformatting. Are these changes required by the "google-java-format" plugin in Default Google Java Style mode?  If so, maybe do formatting in a different PR?

Reply:
Yes this caused by the Google Java Style formatter. We can/should do that in a later PR.